velox/connectors/hive/HiveConnector.h

```diff
@@ -82,7 +84,10 @@ class HiveTableHandle : public ConnectorTableHandle {
     return remainingFilter_;
   }
 
+  std::string toString() const;
```

Comment:
Got an error:

Overrides should be marked explicitly.
toString overrides methods from ConnectorTableHandle but is not annotated with override.